apps/dot_ext/forms.py

```diff
@@ -21,19 +22,23 @@ def __init__(self, user, *args, **kwargs):
         super(CustomRegisterApplicationForm, self).__init__(*args, **kwargs)
         choices = []
         groups = user.groups.values_list('id', flat=True)
+        pcs = None
         for g in groups:
             pcs = ProtectedCapability.objects.filter(group=g)
             for i in pcs:
                 choices.append([i.pk, i.title])
         self.fields['scope'].choices = choices
         self.fields['scope'].label = "Scope*"
+        if pcs:
+            self.fields['scope'].initial = pcs
```

Comment:
it looks like `pcs` will be the result of the last call to `ProtectedCapability.objects.filter(group=g)` is that a problem? For example if there's no `ProtectedCapability` for the last group pcs might be `None` or `[]` and this contitional wouldn't trigger.

Reply:
@whytheplatypus Yes . Good catch. I default it to `None` a few lines up.

Reply:
Right sorry my worry is that durring the for loop it might get set to a real value then get set back to None by a later item `groups`